setup.cfg

```diff
@@ -56,6 +53,8 @@ setup_requires =
 install_requires =
     defusedxml
     keyring
+    urllib3>=2.0.7
+    requests>=2.31.0
```

Comment:
Had to add it manually, otherwise pip-compile for some reason used older version